pyrate/merge.py

```diff
@@ -310,7 +310,7 @@ def __save_merged_files(ifgs_dict, params, array, out_type, index=None, savenpy=
                          f"{ifc.LOS_PROJECTION_OPTION[params[C.LOS_PROJECTION]]} for file {dest}")
             incidence = shared.Geometry(incidence_path)
             incidence.open()
-            array /= los_projection_divisors[params[C.LOS_PROJECTION]](incidence.data)
+            array /= los_projection_divisors[params[C.LOS_PROJECTION]](incidence.data) * params[C.SIGNAL_POLARITY]
```

Comment:
The sign convention is applied in merge, but in the example configs it is listed as a parameter for prepifg

Reply:
I have removed the extra comments from backend test configs, and updated the top level config to show it is applied in MERGE